Column type 168 not implemented

Hi, sorry to bother you again, but I think I may have found another unimplemented feature which is now causing an error.

When I try to load in the attached file, I get "NotImplementedError: column type 168 not implemented".

I am having this same problem with all the data I have recorded today. I didn't update EC-lab or change the parameters of the technique so I find it really odd that the data from today cannot be loaded when all the older data files can.

[files.zip](https://github.com/echemdata/galvani/files/3751331/files.zip)

Ah, that kinda makes sense.

However, I find that adding that line results in a new error 
`AssertionError: 9 columns passed, passed data had 10 columns`

AssertionError: 9 columns passed, passed data had 10 columns

This is seem like an error from pandas not galvani. It seem you might be passing the column names to the Dataframe with out including the the new "Rcmp" column. That's why using the this code is helpfull.

```
def sanitize_labels(label):
    return label.split('/')[0].lower().lstrip('-').translate(dict.fromkeys(map(ord, " ,|()-"),'_')).rstrip('_')

mpr = BioLogic.MPRfile("file_name.mpr")
df = pd.DataFrame(mpr.data)
df.columns = [sanitize_labels(label) for label in df.columns]
df['redox'] = mpr.get_flag('ox/red')
```

--- a/galvani/BioLogic.py
+++ b/galvani/BioLogic.py
@@ -238,7 +238,9 @@ def VMPdata_dtype_from_colIDs(colIDs):
             type_list.append((unique_field_name, field_type))
         else:
             raise NotImplementedError("Column ID {cid} after column {prev} "
-                    "is unknown".format(cid=colID, prev=type_list[-1][0]))
+                                      "is unknown"
+                                      .format(cid=colID,
+                                              prev=type_list[-1][0]))
     return np.dtype(type_list), flags_dict
 
 

--- a/galvani/BioLogic.py
+++ b/galvani/BioLogic.py
@@ -238,7 +238,7 @@ def VMPdata_dtype_from_colIDs(colIDs):
             type_list.append((unique_field_name, field_type))
         else:
             raise NotImplementedError("Column ID {cid} after column {prev} "
-                "is unknown".format(cid=colID, prev=type_list[-1][0]))
+                    "is unknown".format(cid=colID, prev=type_list[-1][0]))
     return np.dtype(type_list), flags_dict
 
 

--- a/galvani/BioLogic.py
+++ b/galvani/BioLogic.py
@@ -166,6 +166,7 @@ VMPdata_colID_dtype_map = {
     125: ('Capacitance charge/µF', '<f8'),
     126: ('Capacitance discharge/µF', '<f8'),
     131: ('Ns', '<u2'),
+    168: ('Rcmp/Ohm', '<f4'),
     169: ('Cs/µF', '<f4'),
     172: ('Cp/µF', '<f4'),
     434: ('(Q-Qo)/C', '<f4'),
@@ -236,6 +237,9 @@ def VMPdata_dtype_from_colIDs(colIDs):
                 unique_field_name = field_name
             type_list.append((unique_field_name, field_type))
         else:
+            print('ColIDs: ',
+                  [colID for colID in colIDs if colID not in VMPdata_colID_flag_map])
+            print('Types: ', type_list[1:])
             raise NotImplementedError("column type %d not implemented" % colID)
     return np.dtype(type_list), flags_dict
 

--- a/galvani/BioLogic.py
+++ b/galvani/BioLogic.py
@@ -237,10 +237,8 @@ def VMPdata_dtype_from_colIDs(colIDs):
                 unique_field_name = field_name
             type_list.append((unique_field_name, field_type))
         else:
-            print('ColIDs: ',
-                  [colID for colID in colIDs if colID not in VMPdata_colID_flag_map])
-            print('Types: ', type_list[1:])
-            raise NotImplementedError("column type %d not implemented" % colID)
+            raise NotImplementedError("Column ID {cid} after column {prev} "
+                "is unknown".format(cid=colID, prev=type_list[-1][0]))
     return np.dtype(type_list), flags_dict
 
 

